modules/smart-agent_systemd-timers/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,11 @@
+module: systemd-timers
```

Comment:
I am not sure if it is a good idea to have an hearbeat in this case, can you confirm the metric reporting is regular and relevant?

Reply:
It is regular, relevant it depends: the alert will be triggered if the service is removed. I think it could be useful, especially on instances which aren't replaced on a regular basis such as a bastion.